docs/_data/menu-sql.yaml

```diff
@@ -61,8 +61,8 @@
 - text: PySpark Usage Guide for Pandas with Apache Arrow
   url: sql-pyspark-pandas-with-arrow.html
   subitems:
-    - text: Apache Arrow in Spark
-      url: sql-pyspark-pandas-with-arrow.html#apache-arrow-in-spark
+    - text: Apache Arrow in PySpark
+      url: sql-pyspark-pandas-with-arrow.html#apache-arrow-in-pyspark
```

Comment:
Actually, can you remove these submenus? It was moved to PySpark documentation after (https://github.com/apache/spark/pull/29548):
![Screen Shot 2020-11-23 at 4 54 55 PM](https://user-images.githubusercontent.com/6477701/99939105-acb37900-2dac-11eb-82ff-26048f541a47.png)

Reply:
Looks great , thanks!